lib/cfme-versions.rb

```diff
@@ -91,7 +91,7 @@ def [](index)
 
       # Version of this gem/tool
       def version
-        versions.last.cfme_release.split(".").select { |val| val =~ /^\d+$/ }.join('.')
+        versions.last.cfme_release.split(".").select { |val| val =~ /^\d+$/ }.join('.') + "-1"
```

Comment:
Used a `-1` here instead of a point release to denote this isn't associated with the CFME version, but we can do a different convention if you want.